Dockerfile.aarch64

```diff
@@ -23,7 +23,7 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM aarch64/ubuntu:trusty
+FROM aarch64/ubuntu:vivid
```

Comment:
When I try with trusty or jessie, I get below error. With vivid this issue is not seen.
Basically apt-get install gccgo installs gccgo-4.9 in trusty & jessie, where as in vivid
gccgo-5 is installed.

Step 13 : RUN update-alternatives --set go /usr/bin/go-5
 ---> Running in bf217edb144c
update-alternatives: error: no alternatives for go
The command '/bin/sh -c update-alternatives --set go /usr/bin/go-5' returned a non-zero code: 2
Makefile:93: recipe for target 'build' failed
make: *** [build] Error 1
ubuntu@ubuntu:~/docker_latest/docker$